virt/post_install/index.adoc

```diff
@@ -0,0 +1,91 @@
+:_content-type: ASSEMBLY
+[id="post-install-configuration-overview"]
+= Post-installation configuration overview
+include::_attributes/common-attributes.adoc[]
+:context: post-installation-configuration-overview
+
+toc::[]
+
+COPIED FROM OCP post-install
```

Comment:
This should probably be commented out even if this is a WIP/not published.

Reply:
Oops. I meant to put that in a separate PR. Will delete!